.github/workflows/test.yml

```
@@ -93,7 +93,7 @@ jobs:
     - name: Docs
       if: startsWith(matrix.os, 'ubuntu') && matrix.python-version == 3.9
       run: |
-        python3 -m pip install sphinx-copybutton sphinx-issues sphinx-removed-in sphinx-rtd-theme sphinxext-opengraph
+        python3 -m pip install sphinx-copybutton sphinx-issues>=3.0.1 sphinx-removed-in sphinx-rtd-theme sphinxext-opengraph
```

```suggestion
        python3 -m pip install sphinx-copybutton sphinx-issues sphinx-removed-in sphinx-rtd-theme sphinxext-opengraph
```
Rerunning main, [it passes now without pinning](https://github.com/python-pillow/Pillow/runs/4781875702), so we don't need to pin the version for GHA.